testsuite/single_node_performance.py

```diff
@@ -320,6 +322,16 @@ class RunGroupConfig:
         (True, [] if FA_MIGRATION_COMPLETE else ["VM", "NativeVM"])
     ]
     for executor_type in executor_types
+] + [
+    RunGroupConfig(
+        expected_tps=7600,
+        key=RunGroupKey("keyless-coin-transfer"),
+        key_extra=RunGroupKeyExtra(
+            txn_auth_mode="keyless",
+            transaction_type_override="non-conflicting-coin-transfer"
```

Comment:
to have a best comparison, let's use no-op or apt-fa-transfer - so we can compare keyless speed to no-keyless directly?

if we use no-op, then we could have only 10k accounts, and have additional_dst_pool_accounts=0? so that might be best?

Reply:
fixed